### DISCUSSION TITLE
Are there any "stopping point" Airflow 2 versions when upgrading from 1.10.15->latest?

### DISCUSSION THREAD

**Discussion by mike-hurley-rally** [apache/airflow#24968] [Q&A]
We're currently on Airflow 1.10.12 and coming up with our upgrade plan to the latest 2.x.

We're planning on doing the upgrade to the bridge version 1.10.15. After we get there, are there any recommended stops within the 2.x series? The only documentation I found was https://airflow.apache.org/docs/apache-airflow/stable/upgrading-from-1-10/index.html which mostly mentions 2.0 and some later versions with some notes, but no updated upgrade plan for later 2.x versions.

Can we go from 1.10.15 straight to latest 2.x? Should we do 1.10.15->2.0.0->latest? Any other "stopping point" versions we should be aware of?

**Comment by potiuk** [comment]
Yes. You should be able to migrate to latest version straight from 1.10.15.
